test/NuGet.Core.Tests/NuGet.Protocol.Tests/PackageSearchResourceV3Tests.cs

```diff
@@ -137,6 +138,7 @@ public async Task PackageSearchResourceV3_GetMetadataAsync_VersionsDownloadCount
             Assert.Equal(28390569, package.DownloadCount);
             Assert.Equal(14, versions.Count());
             Assert.Equal(64099, versions.First().DownloadCount);
+            Assert.Equal(largerThanIntMax, versions.Skip(1).First().DownloadCount);
```

Comment:
Change `versions` to include `.ToList()`, then

```suggestion
            Assert.Equal(14, versions.Count);
            Assert.Equal(64099, versions[0].DownloadCount);
            Assert.Equal(largerThanIntMax, versions[1].DownloadCount);
```

Reply:
Thank you for review. Taken the suggestion.